packages/cli/src/commands/deploy/deployer.ts

```diff
@@ -78,18 +78,28 @@ export abstract class Deployer {
     console.log('Writing files...');
   }
 
-  async deployCommand({ scope, siteId }: { scope: string; siteId?: string }) {
-    console.log(`Deploy command ${scope}...${siteId || ''}`);
+  async deployCommand({ scope, siteId, projectName }: { scope: string; siteId?: string; projectName?: string }) {
+    console.log(`Deploy command ${scope}...${siteId || ''} to ${projectName || '.mastra'}`);
```

Comment:
default project name could be mastra-starter rather than `.mastra`, what do you think?

Reply:
yeah, good point